app-samples/Cognitive-Services-Face-WPF/Sample-WPF/Controls/FaceDetectionPage.xaml.cs

```diff
@@ -198,6 +208,42 @@ public ImageSource SelectedFile
             }
         }
 
+        /// <summary>
+        /// Set whether to draw headpose in face Rect.
```

Comment:
`Sets whether to draw face rectangles rotated by head poses`

Reply:
Done.